ods_ci/tests/Tests/650__distributed_workloads/test-distributed-workloads-metrics-ui.robot

```diff
@@ -0,0 +1,115 @@
+*** Settings ***
+Documentation       Suite to test Workload metrics feature
+Library             SeleniumLibrary
+Library             OpenShiftLibrary
+Resource            ../../Resources/Page/DistributedWorkloads/WorkloadMetricsUI.resource
+Resource            ../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Suite Setup         Project Suite Setup
+Suite Teardown      Project Suite Teardown
+Test Tags           DistributedWorkloadMetrics
+
+
+*** Variables ***
+${PRJ_TITLE}=    test-dw-ui
+${PRJ_TITLE_NONADMIN}=    test-dw-nonadmin
+${PRJ_DESCRIPTION}=    project used for distributed workload metrics
+
+
+*** Test Cases ***
+Verify Workload Metrics Home page Contents
+    [Documentation]    Verifies "Workload Metrics" page is accessible from
+    ...                the navigation menu on the left and page contents
+    [Tags]    Sanity
+    ...       DistributedWorkloads
+    Open Distributed Workload Metrics Home Page
+    Wait until Element is Visible    ${DISTRIBUITED_WORKLOAD_METRICS_TEXT_XP}   timeout=20
+    Page Should Contain Element     ${DISTRIBUITED_WORKLOAD_METRICS_TITLE_XP}
+    Page Should Contain Element     ${DISTRIBUITED_WORKLOAD_METRICS_TEXT_XP}
+    Page Should Contain Element     ${PROJECT_XP}
+    Page Should Contain Element     ${PROJECT_METRICS_TAB_XP}
+    Page Should Contain Element     ${WORKLOAD_STATUS_TAB_XP}
+    Click Element    ${REFRESH_INTERNAL_MENU_XP}
+    ${get_refresh_interval_list}=    Get All Text Under Element   xpath=//*[starts-with(@id, "select-option-")]
+    Lists Should Be Equal    ${REFRESH_INTERNAL_LIST}    ${get_refresh_interval_list}
+
+Verify Project Metrics Default Page contents
+    [Tags]    Sanity
+    ...       DistributedWorkloads
+    [Documentation]    Verifiy Project Metrics default Page contents
+    Open Distributed Workload Metrics Home Page
+    Select Distributed Workload Project By Name    ${PRJ_TITLE}
+    Check Project Metrics Default Page Contents
+
+Verify Distributed Workload status Default Page contents
+    [Tags]    Sanity
+    ...       DistributedWorkloads
+    [Documentation]    Verifiy distributed workload status page default contents
+    Open Distributed Workload Metrics Home Page
+    Select Distributed Workload Project By Name    ${PRJ_TITLE}
+    Check Distributed Workload Status Page Contents
+
+Verify That Not Admin Users Can Access Distributed workload metrics default page contents
+    [Documentation]    Verify That Not Admin Users Can Access Distributed workload metrics default page contents
+    [Tags]    Sanity
```

Comment:
I think that this may be executed as part of Tier 1, no need to run as sanity test.
This test isn't a core basic test.

Reply:
Yes will update the Tags